src/komodo_utils.h

```diff
@@ -2497,3 +2497,8 @@ void komodo_prefetch(FILE *fp)
     }
     fseek(fp,fpos,SEEK_SET);
 }
+
+bool komodo_Is2021JuneHFActive()
+{
+    return GetLatestTimestamp(komodo_currentheight()) > nS5Timestamp;
```

Comment:
any specific reason why we use timestamp as opposed to block height check? As you know for Komodo blockchain it is `komodo hardfork-height != hardfork-timestamp` and difference can be significant

Reply:
I guess timestamp check only is used bcz this change is related to CC, and CC are active only in smartchains for now. But may be will be better to define `komodo_Is2021JuneHFActive` as:
```
bool komodo_Is2021JuneHFActive(uint32_t time)
{
    return ( (ASSETCHAINS_SYMBOL[0] == 0 && chainActive.Height() > nS5HardforkHeight) || (ASSETCHAINS_SYMBOL[0] != 0 && time > nS5Timestamp) );
}
```

Reply:
this fix is supposed for cc chains (maybe it is better to rename the komodo_Is2021JuneHFActive() function to clarify it is timestamp based)

Reply:
this function was designed as self-contained, with no params, as it is called only one-time to fix one specific potential issue in standard.cpp (where we do not have neither chain time nor height)

Reply:
i see. just offered a universal function, that can be used in other places too if needed. bcz currently we haven't any other function to determin is the S5 hardfork active or not.

Reply:
the code is not active on any chains other than ac_cc, so it's safe to do only the timestamp activation IMO